# Y-Chromosome Variation in Southern African Khoe-San Populations Based on Whole-Genome Sequences 

Thijessen Naidoo ${ }^{1,2,3,4, \dagger}$, Jingzi Xu ${ }^{1, \dagger}$, Mário Vicente ${ }^{1}$, Helena Malmström ${ }^{1,5}$, Himla Soodyall ${ }^{6,7,8}$, Mattias Jakobsson ${ }^{1,3,5}$, and Carina M. Schlebusch (D) ${ }^{1,3,5, *}$<br>${ }^{1}$ Human Evolution, Department of Organismal Biology, Evolutionary Biology Centre, Uppsala University, Sweden<br>${ }^{2}$ Department of Archaeology and Classical Studies, Stockholm University, Sweden<br>${ }^{3}$ Science for Life Laboratory, Uppsala, Sweden<br>${ }^{4}$ Centre for Palaeogenetics, Stockholm, Sweden<br>${ }^{5}$ Palaeo-Research Institute, University of Johannesburg, Auckland Park, South Africa<br>${ }^{6}$ Division of Human Genetics, School of Pathology, Faculty of Health Sciences, University of the Witwatersrand, Johannesburg, South Africa<br>${ }^{7}$ National Health Laboratory Service, Johannesburg, South Africa<br>${ }^{8}$ Academy of Science of South Africa<br>*Corresponding author: E-mail: carina.schlebusch@ebc.uu.se.

Accepted: 12 May 2020
${ }^{\dagger}$ These authors contributed equally to this work.
Data deposition: The complete Y-chromosome sequences were deposited on the European Genome Phenome Archive (https://www.ebi.ac.uk/ ega/), accession number EGAS00001004459, and are available for research use under controlled access policies.


#### Abstract

Although the human $Y$ chromosome has effectively shown utility in uncovering facets of human evolution and population histories, the ascertainment bias present in early Y-chromosome variant data sets limited the accuracy of diversity and TMRCA estimates obtained from them. The advent of next-generation sequencing, however, has removed this bias and allowed for the discovery of thousands of new variants for use in improving the Y-chromosome phylogeny and computing estimates that are more accurate. Here, we describe the high-coverage sequencing of the whole Y chromosome in a data set of 19 male Khoe-San individuals in comparison with existing whole Y-chromosome sequence data. Due to the increased resolution, we potentially resolve the source of haplogroup B-P70 in the Khoe-San, and reconcile recently published haplogroup A-M51 data with the most recent version of the ISOGG Y-chromosome phylogeny. Our results also improve the positioning of tentatively placed new branches of the ISOGG Ychromosome phylogeny. The distribution of major Y-chromosome haplogroups in the Khoe-San and other African groups coincide with the emerging picture of African demographic history; with E-M2 linked to the agriculturalist Bantu expansion, E-M35 linked to pastoralist eastern African migrations, B-M112 linked to earlier east-south gene flow, A-M14 linked to shared ancestry with central African rainforest hunter-gatherers, and A-M51 potentially unique to the Khoe-San.


Key words: Y chromosome, next-generation sequencing, haplogroups, Khoe-San, southern Africa.

## Introduction

The male-specific portion of the Y chromosome (MSY) has long been regarded as an effective tool in the study of human evolutionary history (Underhill and Kivisild 2007). It has proved useful mainly due to a lack of recombination along its length, making it the longest
haplotypic block in the human genome (Scozzari et al. 2012); and its paternal mode of inheritance. The transmission of an intact haplotype from father to son, changing only through mutation, preserves a simpler record of its history and allows us to study the male contribution to the shaping of humanity.

[^0]
[^0]:    (c) The Author(s) 2020. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution.

    This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com