<!-- PageNumber="186" -->
<!-- PageHeader="A. Tadokoro et al." -->


Table 1. Bacterial strains

| Strain | Relevant characteristics | Source or reference |
| - | - | - |
| W3110 | wild type | Laboratory stock |
| JE7951 | spr-1 | (7) |
| JE7934 | prcΔ::neo | (7) |
| JE7456 | str xyl argG | Laboratory stock |
| CAG12072 | zgj-203::Tn10 | (26) |
| TOP10 | recA1 araD139 Δ(araA-leu) 7697 | Invitrogen |
| Rev1 | spr-1 nlpI1 | Spontaneously from JE7951 |
| Rev2 | spr-1 nlpI2 | Spontaneously from JE7951 |
| Rev3 | spr-1 nlpI3 | Spontaneously from JE7951 |
| Rev4 | spr-1 nlpI4 | Spontaneously from JE7951 |
| Rev5 | spr-1 nlpI5 | Spontaneously from JE7951 |
| Rev6 | spr-1 nlpI6 | Spontaneously from JE7951 |
| TAR1 | str xyl argG zgj-203::Tn10 | CAG12072 (P1) × JE7456 |
| TAR2 | argG zgj-203::Tn10 | TAR1 (P1) x W3110 |
| TAR3 | spr-1 argG zgj-203::Tn10 | $$\mathrm { T A R 1 } \left( \mathrm { P 1 } \right) \times \mathrm { J E 7 9 5 1 }$$ |
| KM354 | recJ pTP223 gam bet exo | (14) |
| MU2 | nlpI::cm | Derived from KM354 by transformation with linearized nlpI::cm DNA |
| MU24 | nlpI::cm | MU2 (P1) x W3110 |
| MU25 | nlpI1 | $$\mathrm { R e v 1 } \left( \mathrm { P 1 } \right) \times \mathrm { T A R 2 }$$ |
| MU26 | nlpI2 | $$\mathrm { R e v 2 } \left( \mathrm { P 1 } \right) \times \mathrm { T A R 2 }$$ |
| MU27 | nlpI3 | $$\mathrm { R e v 3 } \left( \mathrm { P 1 } \right) \times \mathrm { T A R 2 }$$ |
| MU124 | spr-1 nlpI::cm | $$\mathrm { M U 2 } \left( \mathrm { P 1 } \right) \times \mathrm { J E 7 9 5 1 }$$ |
| MU125 | spr-1 nlpI1 | $$\mathrm { R e v 1 } \left( \mathrm { P 1 } \right) \times \mathrm { T A R 3 }$$ |
| MU126 | spr-1 nlpI2 | $$\mathrm { R e v 2 } \left( \mathrm { P 1 } \right) \times \mathrm { T A R 3 }$$ |
| MU127 | spr-1 nlpI3 | $$\mathrm { R e v 3 } \left( \mathrm { P 1 } \right) \times \mathrm { T A R 3 }$$ |
| PN1 | prcΔ::neo nlpI::cm | $$\mathrm { M U 2 4 } \left( \mathrm { P 1 } \right) \times \mathrm { J E 7 9 3 4 }$$ |
| HSB1 | ibpB::cm | Derived from KM354 by transformation with linearized ibpB::cm DNA |
| HSB2 | ibpB::cm | HSB1 (P1) × W3110 |
| HSB3 | spr-1 ibpB::cm | $$\mathrm { H S B 1 } \left( \mathrm { P 1 } \right) \times \mathrm { J E 7 9 5 1 }$$ |


(underlined), 5'-TGGGCCATGGAGCCTTTTTTGCG-3' and
5'-GTCACCATGGGCTGGTCCGATTCTGCC-3'. The ampli-
fied DNA was digested with NcoI, then inserted into the
same site of pBAD Myc-His C. To construct pNI6 encod-
ing the hexa-histidine-tagged NlpI with the amino-acid
substitution G103D, the mutated nlpI in the chromo-
somal DNA of Rev3 (spr-1 nlpI3) was amplified by PCR
using the primers described above, and the amplified
DNA was digested with the NcoI and inserted into the
same site of pBAD Myc-His C. To construct pNI7 encod-
ing the truncated and hexa-histidine-tagged NlpI, the
fragment containing the truncated nlpI was amplified by
PCR using the forward primer 5'-TGGGCCATGGAGC-
CTTTTTTGCG-3' and the reverse primer 5'-GGTCCAT-
GGGGTCCTGGCCC-3'. The amplified DNA was digested
with NcoI, then inserted into the same site of pBAD Myc-
His C. pNI8 and pNI9 were constructed in the same way
as pNI7 except that 5'-CATCCATGGCCTGGCCCAGG-3'
or 5'-CTTCCATGGGGCCCAGGAGC-3', respectively, was
used as the reverse primer in PCR. To construct pNI10
encoding NlpI, the nlpI in the chromosomal DNA of
W3110 was amplified by PCR using the forward primer
containing the NcoI site (underlined) 5'-TGGGCCAT-
GGAGCCTTTTTTGCG-3' and the reverse primer con-
taining the BamHI site (underlined) 5'-TGGATCCGGCT-
CAAAGTAGG-3'. The amplified DNA was digested with
NcoI and BamHI and inserted into the same sites of
pTrc99A. pNI12, pNI13, and pNI14 were constructed in
the same way as pNI10 except that 5'-CCAGGGATCCT-
TAGTCCTGGCC-3', 5'-GTCAGGATCCTACTGGCCCAG-

3', or 5'-ATCTGGATCCTAGCCCAGGAG-3', respectively,
was used as the reverse primer containing the BamHI
site (underlined) in PCR. The amplified DNA was
digested with NcoI and BamHI and then inserted into
the same sites of pTrc99A.

Expression and Purification of Hexa-Histidine-Tagged
NlpI Protein-E. coli cells were transformed with the
recombinant plasmids. The transformants were grown in
L broth at 42°C for 4 h, then incubated in L broth con-
taining 0.01% arabinose at 42°C for 4 h for expression of
the fusion genes. Cells from 400 ml of medium were col-
lected, washed once with ice-cold 50 mM sodium phos-
phate buffer (pH 8.0) containing 300 mM NaCl (buffer A),
suspended in 4 ml of the buffer A containing 10 mM imi-
dazole and 1% Triton X-100, and disrupted by sonication
with cooling by ice-water. The mixture was centrifuged
for 15 min at 17,700 xg to obtain a clear lysate. Four mil-
liliters of the supernatant was mixed with 1 ml of nickel-
nitrilotriacetic acid (Ni-NTA) resin and incubated with
rotation for 2 h at 4°C. The mixture was loaded onto an
open column. The column was washed with 4 ml each of
buffer A containing 20 mM imidazole and then contain-
ing 40 mM imidazole. The bound proteins were eluted
with 1.2 ml of buffer A containing 100 mM imidazole, and
fractions of 0.4 ml were collected.

Mass Spectrometry Analysis-An electrospray ion trap
mass spectrometer (LCQ DECA XP ion trap mass sys-
tem, Thermo Electron) coupled online with a capillary
HPLC (Magic 2002, Michrom BioResources) was used to
acquire MS/MS spectra. A 0.2 x 50 mm MAGICMS C18

<!-- PageFooter="J. Biochem." -->
<!-- PageFooter="Downloaded from http://jb.oxfordjournals.org/ at University of North Dakota on June 15, 2015" -->
```